pending_review/review.go

```diff
@@ -25,11 +25,17 @@ type Reviews struct {
 	Blockers  []string   // List of Conan team members who have requested changes on the head commit
 
 	LastReview *Review `json:",omitempty"` // Snapshot of the last review
+
+	IsBump			bool // PR is a bump of version or dependencies
 }
 
 // IsApproved when the conditions for merging are meet as per https://github.com/conan-io/conan-center-index/blob/master/docs/review_process.md
 func (r *Reviews) IsApproved() bool {
-	return r.TeamApproval && r.ValidApprovals >= 2 && len(r.Blockers) == 0
+	var nRequiredApprovals = 2
+	if r.IsBump {
+		nRequiredApprovals = 1
+	}
+	return r.TeamApproval && r.ValidApprovals >= nRequiredApprovals && len(r.Blockers) == 0
```

Comment:
Do dump PR need to be accepted by a team member or  are community members counted too?

Reply:
bump PRs need a team member approval cf https://github.com/conan-io/conan-center-index/blob/master/docs/review_process.md#rule-of-2-reviews

> Pull requests labelled as [Bump version](https://github.com/conan-io/conan-center-index/pulls?q=is%3Aopen+is%3Apr+label%3A%22Bump+version%22) or [Bump dependencies](https://github.com/conan-io/conan-center-index/pulls?q=is%3Aopen+is%3Apr+label%3A%22Bump+dependencies%22+) require just 1 approving review from an official reviewer. Community reviewers are not required but are still welcome.